skops/saving_utils/model_card.py

```diff
@@ -0,0 +1,55 @@
+from modelcards import CardData, ModelCard
+from sklearn import set_config
+from sklearn.utils import estimator_html_repr
+
+
+def _plot_estimator(model):
+    set_config(display="diagram")
+    return estimator_html_repr(model)
```

Comment:
Do you need to set the config when you're explicitly calling this function anyway?

Reply:
OK apparently that's what it does anyway, I'll directly return that :)

Reply:
do you think it's good to give the people the option to have it ask JSON? @adrinjalali

Reply:
yeah that certainly wouldn't hurt.